gatsby-config.js

```diff
@@ -70,7 +70,13 @@ const plugins = [
     options: {
       dsn: process.env.SENTRY_DSN,
       environment: process.env.NODE_ENV,
-      enabled: process.env.NODE_ENV === 'production'
+      release: process.env.CIRCLE_SHA1,
+      enabled: process.env.NODE_ENV === 'production',
+      ignoreErrors: [
+        // Ignore errors on missing chunks when new version was deployed
```

Comment:
Please, let's put a better comment here - it's not clear why do we have this.

Reply:
Updated comment. Now it should me more clear. Wdyt?